Include/pytime.h

```diff
@@ -29,7 +29,10 @@ typedef enum {
     _PyTime_ROUND_CEILING=1,
```

Comment:
It seems like you can removed _PyTime_ROUND_CEILING: https://bugs.python.org/issue31786#msg304456

Reply:
Oops, my pending PR 3983 uses _PyTime_ROUND_CEILING :-) Please keep it.